### PR TITLE
Initial work on readme.md

### PR DESCRIPTION
**Pull request by peterNordin**
Testing require review for merge

### CODE REVIEW

**Comment by peterNordin** [comment]
Looks good (test comment)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Hopsan
+
+Hopsan is a free open-source multi-domain system simulation tool developed at the division of Fluid and mechatronic systems at Linkoping university.
+
+Features include:
+
+* Simulation core library
+    * Plain C++ library for easy integration
+    * Multi-core support for faster simulations
+    * Create your own component models libraries in C++
```
Comment (marthoch) [inline README.md]: The two lines became one and are then hard to understand.  Two sentences is better.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    * Drag and drop / power-port based modeling including support for reusable subsystems
+    * Advanced simulation result analysis capabilities
+    * Energy losses calculations 
+    * Data export to CSV, XML, Gnuplot, HDF5 & Matlab formats 
+    * Interactive animation of the simulated system (real-time and playback)   
+    * Model variable sensitivity analysis
+    * Frequency-domain analysis (based on simulation results)
+    * Numerical optimization
+    * Scripting using the HCOM or Python language
+    * Model import/export using Functional Mock-Up Interface (FMI)
```
Comment (marthoch) [inline README.md]: The term FMU would make it easier for search engines, as it is a common term.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
+# Hopsan
```
Comment (marthoch) [inline README.md]: A hint to the compiled releases may be helpful. But that would be only temporary.

File: README.md (hunk 1 above)
Comment (peterNordin) [inline README.md]: Yes line breaks are tricky. Apparently the line should end in two white spaces which is easy to forget

File: README.md (hunk 3 above)
Comment (peterNordin) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (peterNordin) [inline README.md]: I make sure to mention both FMI and FMU

**Pull request merged**